bin/rcm-release-data.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+import os
+import sys
+
+
+here = sys.path[0]
+if here not in ('/usr/bin', '/bin', '/usr/local/bin'):
+    # git checkout
+    toplevel = os.path.dirname(here)
+    sys.path[0] = toplevel
+
+from distutils.sysconfig import get_python_lib
+
+sys.path.insert(1, os.path.join(get_python_lib(), "rcm_release"))
+
+from rcm_release import *
```

Comment:
Since this script is pretty simple, we can probably just explicitly spell out the imports here. It will make it much easier to understand.